torchci/clickhouse_queries/torchbench_userbenchmark_list_commits/query.sql

```diff
@@ -1,14 +1,29 @@
--- !!! Query is not converted to CH syntax yet.  Delete this line when it gets converted
 WITH w AS (
-  SELECT ARBITRARY(name) AS name, "torchbench-userbenchmark".environ.pytorch_git_version as pytorch_git_version,
-  ARBITRARY("torchbench-userbenchmark".environ.pytorch_version) as pytorch_version,
-  FROM torchbench."torchbench-userbenchmark"
-  WHERE name = :userbenchmark
-  GROUP BY "torchbench-userbenchmark".environ.pytorch_git_version
+    SELECT
+        any(name) as name,
+        JSONExtractString(environ, 'pytorch_git_version') as pytorch_git_version,
+        any(JSONExtractString(environ, 'pytorch_version')) as pytorch_version
+    FROM
+        benchmark.torchbench_userbenchmark
+    WHERE
+        benchmark.torchbench_userbenchmark.name = {userbenchmark: String }
+    GROUP BY
+        pytorch_git_version
 ),
 s AS (
-  SELECT push._event_time as pytorch_commit_time, push.head_commit.id as sha from push
+    SELECT
+        push.head_commit.timestamp as pytorch_commit_time,
+        push.head_commit.id as sha
+    from
+        default .push
```

Comment:
Not sure if we need to use `final` here, but if we do, I think something like `select DISTINCT push.head_commit.id` would also work.

Reply:
I don't think we need `final` here. I tested some random parameters Rockset/CH, and got matching results without final.